conf/zapi/cdot/9.8.0/volume.yaml

```diff
@@ -64,9 +65,9 @@ counters:
       - snapshot-count
 
 plugins:
-  Snapshot:
-    objects:
-      - total-schedules
+  SnapmirrorVolume:
```

Comment:
I think we use SnapMirror in other places

Reply:
changed to volume zapi plugin